docs/en/resource/introduction.md

```diff
@@ -0,0 +1,101 @@
+# Resource
+
+Dojo **resource** provides a consistent means to make widgets data aware.
+
+Dojo resource allows you to create a single store of data which can be passed into multiple widgets. It allows data to be cached, transformed and filtered to suit the needs of the widget it's using. Coupled with the data middleware, resources allow consistent, source agnostic data access for your widgets without them being aware of the underlying data fetching implementation or the raw data format being used.
+
+| Feature                                   | Description                                                                                                                                                                   |
+| ----------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Single data source**                    | Resources allows you to create a single resource for a given template and share it with multiple widgets via the data middleware.                                             |
+| **Data transforms**                       | A data tranform mechanism allows you to specify the data form at your widget requires and transforms the data transparently                                                   |
+| **Support for async and sync data reads** | The resource template can read data in any way it sees fit, the data middleware uses a reactive API to invalidate widgets when data becomes available.                        |
+| **Consistent Resource Options**           | Resource options objects are passed to all api functions ensuring that all api functions are pure and provide only the data that was requested                                |
+| **Sharable Resource Options**             | Resource options can be shared between widgets via the data middleware meaning that multiple widgets can react to changes to the resource options, ie. query, page number etc |
+
+# Basic Usage
+
+Dojo resource is comprised of a resource which you create to get and read data and a data middleware which you use within your widgets to access the resource.
+
```

Comment:
I think it would be useful to see the definition for a `DataTemplate` for an overview of what fields are available/optional and what they're types are. Obviously, this is information that could be found, but I reckon it's small enough to reference here.

Reply:
It's covered in the next paragraph, do you think that's sufficient? I didnt want to go over the template in the very first section.